spec/features/preassembly_image_accessioning_spec.rb

```diff
@@ -82,6 +82,8 @@
     wait_for_download
     yaml = YAML.load_file(download)
     expect(yaml[:status]).to eq 'success'
+    # delete the downloaded YAML file, so we don't pick it up by mistake during the re-accession
+    delete_download(download)
 
```

Comment:
i noticed the way we look for downloaded YAML files was just picking the first, yet in a re-accession we will have two until we cleanup ... so safer to cleanup the first after it's done before we start the re-accession to avoid false positives